lib/auth/auth_with_roles.go

```diff
@@ -552,6 +556,22 @@ func (a *AuthWithRoles) GetAuthServers() ([]services.Server, error) {
 	return a.authServer.GetAuthServers()
 }
 
+// DeleteAllAuthServers deletes all auth servers
+func (a *AuthWithRoles) DeleteAllAuthServers() error {
+	if err := a.action(defaults.Namespace, services.KindAuthServer, services.VerbDelete); err != nil {
```

Comment:
Random thought: I wonder if "delete all" kind of functions should also check for "list" permission.